build-docker-images.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-KUZZLE_LATEST_MAJOR=1
+KUZZLE_LATEST_MAJOR=$(cat package.json | grep version | sed 's/[\",]//g' | cut -d: -f 2 | cut -d. -f 1)
```

Comment:
Despite the fact that I like your oneliner, the KUZZLE_LATEST_MAJOR must be set by hand because if the latest major is `2` and we release `1.42.0` this will push `1.42.0` as `latest` or the latest version is something like `2.x.x`